openmdao/core/group.py

```diff
@@ -176,8 +176,12 @@ def connect(self, source, targets, src_indices=None):
             If specified, connect the specified entries of the full
             distributed source value to the target.
         """
-        if isinstance(targets, str):
-            targets = (targets,)
+        src_indices = kwargs.pop('src_indices', None)
+        if len(kwargs) > 0:
+            raise TypeError("connect() got an unexpected keyword argument '{0}'".format(kwargs.popitem()[0]))
+
+        if len(targets) == 1 and not isinstance(targets[0], str):
```

Comment:
what if the user does connect('src', 'target', 'Robert'); DROP TABLE STUDENTS; --')